login/login.go

```diff
@@ -34,7 +34,7 @@ var ArgHelp = `
     [login-target]
         The login target can be one of the following: 
 			https://login.bp.nbis.se/
-			https://login.test.fega.nbis.se/
+			https://login.fega.nbis.se/
```

Comment:
remove all references to specific sites.

Reply:
I fixed this even though it still shows up, remote is up to date

Reply:
Is this coming from the two dot vs three dot diff that github does?
https://github.com/NBISweden/sda-cli/blob/264-fix-login-uris/login/login.go

Reply:
No.
All URL needs to be removed.